src/App.js

```diff
@@ -1,7 +1,70 @@
 import Background from "./assets/Background.svg";
 import Logo from "./assets/spacecenter1.svg";
+import { useNavigate } from "react-router-dom";
 
 function App() {
+  const LOGIN_API_URL = "http://localhost:3000/auth/login";
+
+  const CREATE_USER_URL = "http://localhost:3000/users";
+
+  const navigate = useNavigate();
+
+  async function user(mail, pass) {
+    const response = await fetch(CREATE_USER_URL, {
+      method: "POST",
+      body: JSON.stringify({
+        email: mail,
+        password: pass,
+      }),
+      headers: {
+        "Content-type": "application/json; charset=UTF-8",
+      },
+    });
+    const json = await response.json();
+    return json;
+  }
+
+  async function login(mail, pass) {
+    const response = await fetch(LOGIN_API_URL, {
+      method: "POST",
+      body: JSON.stringify({
+        email: mail,
+        password: pass,
+      }),
+      headers: {
+        "Content-type": "application/json; charset=UTF-8",
+      },
+    });
+    const json = await response.json();
+    return json;
+  }
+
+  const handleSubmit = async () => {
+    const mail = document.getElementById("email").value;
```

Comment:
This works, but it gets messy as you add more fields, and have to add ids to everything to get the values. The more correct way of getting form values is using [FormData](https://developer.mozilla.org/en-US/docs/Web/API/FormData/get). You create a new FormData using the event target that this function receives, and then can access all the values in the form, without the need to set ids.